lasagne/layers/shape.py

```diff
@@ -98,8 +95,8 @@ def get_output_shape_for(self, input_shape, *args, **kwargs):
                                      (o[0], len(input_shape)))
                 output_shape[dim] = input_shape[o[0]]
                 masked_output_shape[dim] = input_shape[o[0]]
-                if ((input_shape[o[0]] is None)
-                    and (masked_input_shape[o[0]] is None)):
+                if (input_shape[o[0]] is None) \
+                   and (masked_input_shape[o[0]] is None):
```

Comment:
Me too, but it's because this is bad:
```
if (something == foo and
    something_else == bar):
    this_line = has_the_same_indent_level
```
This, I think, is the only place I use line continuation.